plugins/node/opentelemetry-instrumentation-redis/src/instrumentation.ts

```diff
@@ -29,6 +29,10 @@ import {
 import { RedisInstrumentationConfig } from './types';
 import { VERSION } from './version';
 
+const DEFAULT_CONFIG: RedisInstrumentationConfig = {
+  requireParentSpan: false,
```

Comment:
It would be best to have the same default behaviour with ioredis.
Even though I also think it should, in fact, be `false`.

Reply:
I generally agree but that would mean a change of behavior for existing users of the instrumentation. It could appear as a breaking change in the next release (for either redis or io-redis)

Reply:
Perhaps now, before the GA, is the time to make the breaking changes in those cases. Not in this PR ofc.

Reply:
I'll take care of it (changing the default config for the io-redis instrumentation, as you suggested) in another PR once we merge this one.